config/config-mp/src/main/java/io/helidon/config/mp/MpConfigImpl.java

```diff
@@ -313,35 +334,47 @@ private <T> T convert(String propertyName, Class<T> type, String value) {
         }
     }
 
-    private Optional<ConfigValue> findConfigValue(String propertyName) {
+    private Optional<ConfigValue> findConfigValue(String propertyName, Optional<String> profiledPropertyName) {
```

Comment:
we do not use optional parameters. Also it does not make sense in this case, as you wrap it just to unwrap it later. Please use `null` here, as it is a private method (`null` is not allowed in public API only).

Reply:
Done